modules/social_features/social_core/social_core.install

```diff
@@ -418,3 +417,17 @@ function social_core_update_124001(): void {
   $config = \Drupal::service('config.factory')->getEditable('lazy.settings');
   $config->delete()->save();
 }
+
+/**
+ * Unset permission Administer vocabularies and terms from the site manager.
+ */
+function social_core_update_130001(): string {
+  /** @var \Drupal\update_helper\Updater $updateHelper */
```

Comment:
@Macsym2018 We don't  need to use `update_helper` in this case and can simply use the following code in the update hook:

```
user_role_revoke_permissions('sitemanager', ['administer taxonomy']);
```